ci/__main__.py

```diff
@@ -0,0 +1,686 @@
+
+# Location of the Changelog file.
+CHANGELOG_PATH = os.path.join(HERE, "CHANGES.txt")
+
+
+def normalize(name):
+    return name.replace("_", "-")
+
+
+# Ensure that "-h" is supported for getting help.
+CONTEXT_SETTINGS = dict(
+    help_option_names=["-h", "--help"],
+    token_normalize_func=normalize,
+)
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+def cli():
+    pass
+
+
+@cli.command()
+@click.option('--runtime', default=DEFAULT_RUNTIME)
+@click.option('--toolkit', default=DEFAULT_TOOLKIT)
+@click.option('--environment', default=None)
+@click.option(
+    "--editable/--not-editable",
+    default=False,
+    help="Install main package in 'editable' mode?  [default: --not-editable]",
+)
+@click.option('--source/--no-source', default=False)
+def install(runtime, toolkit, environment, editable, source):
+    """Install project and dependencies into a clean EDM environment."""
+    parameters = get_parameters(runtime, toolkit, environment)
+    packages = (dependencies | ci_dependencies)
+
+    # edm commands to setup the development environment
+    commands = [
+        "edm environments create {environment} --force --version={runtime}"
+    ]
+
+    commands.extend(
+        [
+            "edm install -y -e {environment} " + " ".join(packages),
+            # "edm run -e {environment} -- pip install --force-reinstall -r
+            # ci-src-requirements.txt --no-dependencies",
+            "edm run -e {environment} -- python setup.py clean --all",
+        ]
+    )
+
+    # pip install pyqt5, pyqt6, pyside2 and pyside6, because we don't have them
+    # in EDM yet
+    if toolkit == 'pyside2':
+        commands.append("edm run -e {environment} -- pip install pyside2")
+    elif toolkit == 'pyqt6':
+        commands.append("edm run -e {environment} -- pip install pyqt6")
+    elif toolkit == 'pyside6':
+        # On Linux and macOS, some versions of PySide6 between 6.2.2 and 6.3.0
+        # are unimportable on Python 3.6 and Python 3.7. See
+        # https://bugreports.qt.io/browse/PYSIDE-1797. It may be possible to
+        # remove this workaround once PySide6 6.3.1 or later is released.
+        if sys.platform in {'darwin', 'linux'}:
+            commands.append(
+                "edm run -e {environment} -- pip install pyside6<6.2.2")
+        else:
+            commands.append("edm run -e {environment} -- pip install pyside6")
+    elif toolkit == 'wx':
+        if sys.platform != 'linux':
+            commands.append("edm run -e {environment} -- pip install wxPython")
+        else:
+            # XXX this is mainly for TravisCI workers; need a generic solution
+            commands.append(
+                "edm run -e {environment} -- pip install -f "
+                "https://extras.wxpython.org/wxPython4/extras/linux/gtk3"
+                "/ubuntu-16.04/ wxPython"
+            )
+
+    click.echo("Creating environment '{environment}'".format(**parameters))
+    execute(commands, parameters)
+
+    if source:
```

Comment:
We don't need this `if` block either as we don't want to install any packages from source.